lib/player.js

```diff
@@ -3520,6 +3520,19 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
                        t.label != shaka.Player.TextTrackLabel);
   }
 
+  /**
+   * Get the TextTracks with the 'metadata' kind.
+   *
+   * @return {!Array.<TextTrack>}
+   * @private
+   */
+  filterMetadataTracks_() {
```

Comment:
I suggest naming this `getMetadataTracks_`.  Maybe you should also rename the other to `getFilteredTextTracks_`.

Reply:
Done!